src/main/java/core/basesyntax/figures/Circle.java

```diff
@@ -0,0 +1,34 @@
+package core.basesyntax.figures;
+
+import core.basesyntax.enums.Color;
+
+public class Circle extends Figure {
+    private double radius;
+
+    public Circle(Color color, double radius) {
+        super(color);
+        this.radius = radius;
+    }
+
+    public double getRadius() {
+        return radius;
+    }
+
+    public void setRadius(double radius) {
+        this.radius = radius;
+    }
+
+    @Override
+    public double calculateArea() {
+        return 2 * Math.pow(radius, 2) * Math.PI;
```

Comment:
The formula for calculating the area of a circle is incorrect. It should be `Math.PI * Math.pow(radius, 2)` instead of `2 * Math.pow(radius, 2) * Math.PI`. Please correct this to ensure the area is calculated properly.